api/app/users/models.py

```diff
@@ -11,16 +11,64 @@ def expiration_date():
 class AppUser(db.Model):
 
     id = db.Column(db.Integer(), primary_key=True)
-    email = db.Column(db.String(255), unique=True)
-    password = db.Column(db.String(255))
-    active = db.Column(db.Boolean())
-    is_admin = db.Column(db.Boolean())
+    email = db.Column(db.String(255), unique=True, nullable=False)
+    firstname = db.Column(db.String(100), nullable=False)
+    lastname = db.Column(db.String(100), nullable=False)
+    user_title_id = db.Column(db.Integer(), db.ForeignKey('user_title.id'), nullable=False)
+    nationality_country_id = db.Column(db.Integer(), db.ForeignKey('country.id'), nullable=False)
+    residence_country_id = db.Column(db.Integer(), db.ForeignKey('country.id'), nullable=False)
+    user_ethnicity_id = db.Column(db.Integer(), db.ForeignKey('user_ethnicity.id'), nullable=False)
```

Comment:
I still see id's. Can we drop gender_id, ethnicity_id, disability_id, and user_title id?

Reply:
Do you mean it should be strings and we shouldn't have tables for the options?

Because I disagree on that approach. I think it will make the reporting and filtering easier. We reduce the chance of any data integrity issues and it allows the options to be configurable without a redeploy.

Reply:
Sure. It probably is a over-normalised. I'll update.

Reply:
@jaderabbit Changes have been pushed. Can you also please check the comment I made on `testing.py` file? I'm not sure what I did there was correct.

Reply:
Clearing up the db before each test is perfect :)

Reply:
@jaderabbit Thanks for checking.

And woohoo, my first ever PR to be accepted to an open source project :)

Reply:
@amritpurshotam Congrats! It's always a tremendous feeling :smile: